pulpcore/cli/common/__init__.py

```diff
@@ -117,6 +118,7 @@ def main(
     refresh_api: bool,
     dry_run: bool,
     timeout: int,
+    cid: str,
```

Comment:
I recently learned that this behaviour may be changed by settings that we hand over to django-guid. So we cannot assume that it _must_ be a uuid.
Only thing we can try to do is to warn the user if the server ignored our cid.

Reply:
We def need to have something here - I tried the PR, and my first thought was "this functionality is broken" because I didn't realize the server could say "no"